torch/_tensor.py

```diff
@@ -1327,23 +1328,22 @@ def __dlpack__(self, stream=None):
         return torch.to_dlpack(self)
 
     def __dlpack_device__(self) -> Tuple[enum.IntEnum, int]:
-        # Avoid circular import
-        from torch.utils.dlpack import DLDeviceType
```

Comment:
It looks like the import was previously put here for a reason: "Avoid circular import". Is  that no longer a concern?

Reply:
The change to file `torch/utils/dlpack.py` is intended to resolve the circular import issue.